ci/tox/Dockerfile

```diff
@@ -29,15 +29,17 @@ ENV PYTHON_VERSION=${python_version}
 ENV PYTHON=python${python_version}
 RUN apk add --no-cache gcc musl-dev # both packages are required to install mypy
 
-ARG user_id
-ARG group_id
+# ARG user_id
+# ARG group_id
 # RUN set -x \
 #     && [ ${user_id:-0} -ne 0 ] \
 #     && [ ${group_id:-0} -ne 0 ] \
 #     # sometimes the given `group_id` is already taken and `addgroup` raises an error, so let's check its existence first
 #     && (getent group $group_id || addgroup --gid=${group_id} ci-user) \
 #     && adduser --uid=${user_id} --ingroup=$(getent group $group_id | cut -d: -f1) --disabled-password ci-user
 # USER ci-user
+ENV USER_NAME=${id -un}
```

Comment:
Whoops `ENV`s in Dockerfile are unforunately unable of performing _command substitution_ aka `$(...)` :/